### ISSUE TITLE
When I pass svg including Japanese String to SvgImage, an exception will occur.

### ISSUE THREAD

**Issue by prestojapan2021** [DavBfr/dart_pdf#1092]
**Describe the bug**
An error occurs when Japanese is added to document.svg and executed by the demo program.
Error display on DOCUMET TAB is 

Invalid argument (string):
Contains invalid charactore/: "日本語"

**To Reproduce**
Document.svg:
```svg string
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   viewBox="0 0 150 230"
   version="1.1"
   id="svg16"
   sodipodi:docname="document.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <sodipodi:namedview
     id="namedview18"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="3.5826087"
     inkscape:cx="72.851942"
     inkscape:cy="115"
     inkscape:window-width="1432"
     inkscape:window-height="1032"
     inkscape:window-x="180"
     inkscape:window-y="0"
     inkscape:window-maximized="0"
     inkscape:current-layer="svg16" />
  <defs
     id="defs8">
    <linearGradient
       id="gradient-1">
      <stop
         offset="0"
         style="stop-color: rgb(255, 255, 255);"
         id="stop2" />
      <stop
         offset="1"
         style="stop-color: rgb(161, 161, 161);"
         id="stop4" />
    </linearGradient>
    <linearGradient
       id="gradient-1-0"
       gradientUnits="userSpaceOnUse"
       x1="119.315"
       y1="36.313"
       x2="119.315"
       y2="216.359"
       xlink:href="#gradient-1" />
  </defs>
  <rect
     x="-67.203"
     y="-48.72"
     width="276.134"
     height="354.501"
     style="fill: rgb(93, 32, 205);"
     id="rect10" />
  <ellipse
     style="fill: rgb(141, 81, 252);"
     cx="119.315"
     cy="126.336"
     rx="139.238"
     ry="98.673"
     transform="matrix(0.93642, 0.35088, -0.35088, 0.93642, -15.610807, -59.255697)"
     id="ellipse12" />
  <ellipse
     style="fill: url(#gradient-1-0);"
     cx="119.315"
     cy="126.336"
     rx="130.398"
     ry="90.023"
     transform="matrix(0.952861, 0.303408, -0.303408, 0.952861, -26.924128, -56.954295)"
     id="ellipse14" />
  <text
     xml:space="preserve"
     style="font-style:normal;font-weight:normal;font-size:40px;line-height:1.25;font-family:sans-serif;fill:#000000;fill-opacity:1;stroke:none"
     x="-11.165049"
     y="82.621361"
     id="text26757"><tspan
       sodipodi:role="line"
       id="tspan26755"
       x="-11.165049"
       y="82.621361"
       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-family:'MS Mincho';-inkscape-font-specification:'MS Mincho'">日本語</tspan></text>
</svg>

Launching lib\main.dart on sdk gphone64 x86 64 in debug mode...
Running Gradle task 'assembleDebug'...
√  Built build\app\outputs\flutter-apk\app-debug.apk.
Installing build\app\outputs\flutter-apk\app.apk...
Debug service listening on ws://127.0.0.1:59718/kUXO2bN_4sE=/ws
Syncing files to device sdk gphone64 x86 64...
D/EGL_emulation(26599): app_time_stats: avg=37.15ms min=4.66ms max=671.86ms count=32
D/EGL_emulation(26599): app_time_stats: avg=7.52ms min=5.27ms max=16.50ms count=32
D/EGL_emulation(26599): app_time_stats: avg=6.88ms min=4.79ms max=10.24ms count=31
D/EGL_emulation(26599): app_time_stats: avg=8.33ms min=4.57ms max=14.32ms count=32
D/EGL_emulation(26599): app_time_stats: avg=7.20ms min=4.81ms max=10.43ms count=32
D/EGL_emulation(26599): app_time_stats: avg=6.74ms min=4.69ms max=15.49ms count=31
D/EGL_emulation(26599): app_time_stats: avg=7.39ms min=4.91ms max=19.22ms count=32
D/EGL_emulation(26599): app_time_stats: avg=7.02ms min=4.78ms max=15.70ms count=32
D/EGL_emulation(26599): app_time_stats: avg=6.19ms min=4.39ms max=8.31ms count=31
D/EGL_emulation(26599): app_time_stats: avg=6.50ms min=4.03ms max=15.48ms count=31
D/EGL_emulation(26599): app_time_stats: avg=6.87ms min=4.69ms max=19.09ms count=31
D/EGL_emulation(26599): app_time_stats: avg=21.19ms min=4.68ms max=436.00ms count=30
D/EGL_emulation(26599): app_time_stats: avg=1041.99ms min=5.45ms max=15272.35ms count=15
I/flutter (26599): Helvetica has no Unicode support see https://github.com/DavBfr/dart_pdf/wiki/Fonts-Management
I/flutter (26599): ---------------------------------------------
I/flutter (26599): Cannot decode the string to Latin1.
I/flutter (26599): This font does not support Unicode characters.
I/flutter (26599): If you want to use strings other than Latin strings, use a TrueType (TTF) font instead.
I/flutter (26599): See https://github.com/DavBfr/dart_pdf/wiki/Fonts-Management
I/flutter (26599): ---------------------------------------------

======== Exception caught by printing ==============================================================
The following ArgumentError was thrown while generating a PDF:
Invalid argument (string): Contains invalid characters.: "日本語"

When the exception was thrown, this was the stack: 
#0      _UnicodeSubsetEncoder.convert (dart:convert/ascii.dart:85:9)
#1      Latin1Codec.encode (dart:convert/latin1.dart:40:46)
#2      PdfFont.stringMetrics (package:pdf/src/pdf/obj/font.dart:174:28)
#3      new SvgText.fromXml (package:pdf/src/svg/text.dart:70:29)
#4      new SvgText.fromXml.<anonymous closure> (package:pdf/src/svg/text.dart:87:29)
#5      MappedIterator.moveNext (dart:_internal/iterable.dart:391:20)
#6      SvgText.paintShape (package:pdf/src/svg/text.dart:159:24)
#7      SvgOperation.paint (package:pdf/src/svg/operation.dart:98:5)
#8      SvgGroup.paintShape (package:pdf/src/svg/group.dart:62:13)
#9      SvgOperation.paint (package:pdf/src/svg/operation.dart:98:5)
#10     SvgPainter.paint (package:pdf/src/svg/painter.dart:46:48)
#11     SvgImage.paint (package:pdf/src/widgets/svg.dart:130:13)
#12     Page.paint (package:pdf/src/widgets/page.dart:232:13)
#13     Page.postProcess (package:pdf/src/widgets/page.dart:173:7)
#14     Document.save (package:pdf/src/widgets/document.dart:131:14)
#15     generateDocument (package:printing_demo/examples/document.dart:286:20)
<asynchronous suspension>
#16     PdfPreviewRaster._raster (package:printing/src/preview/raster.dart:124:14)
<asynchronous suspension>
PageFormat: "PdfPageFormat 612.0x792.0 margins:72.0, 72.0, 72.0, 72.0"
====================================================================================================
D/EGL_emulation(26599): app_time_stats: avg=514.06ms min=6.37ms max=6462.59ms count=13

```

**Expected behavior**
document.svg is displayed in the background.

**Screenshots**

![error](https://user-images.githubusercontent.com/36428714/176799903-f5520837-12cb-43e8-aa2c-f37bfa46239e.png)

**Flutter Doctor**

```
PS C:\Users\sugiy\Desktop\dart_pdf-master\dart_pdf-master\demo> flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 3.0.3, on Microsoft Windows [Version 10.0.22000.739], locale ja-JP)
[√] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
[√] Chrome - develop for the web
[√] Visual Studio - develop for Windows (Visual Studio Build Tools 2019 16.11.9)
[√] Android Studio (version 2021.2)
[√] VS Code (version 1.68.1)
[√] Connected device (4 available)
[√] HTTP Host Availability

• No issues found!

```

**Desktop (please complete the following information):**
 - [ ] iOS
 - [*] Android
 - [ ] Browser
 - [ ] Windows
 - [ ] Linux

**Smartphone (please complete the following information):**
 - Device: Virtual Pixel 2 (Android 12.0 Google Play | x86_64)

**Additional context**

**Comment by prestojapan2021** [comment]
i am sorry, This exception is created by PdfPreview().
This mehod is included in  printing: ^5.9.1.

**Comment by DavBfr** [comment]
Yes, text rendering is bare minimum on the SVG.
You should convert your text to paths with a drawing software, like Inkscape.
